packages/b/brogue/files/io.github.tmewett.brogue.metainfo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.tmewett.brogue</id>
+  
+  <name>Brogue</name>
+  <summary>Brogue is a Roguelike game by Brian Walker (community edition)</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-3.0-or-later</project_license>
+  
+  <description>
+    <p>
+      Brogue is a Roguelike game by Brian Walker. It is a direct descendant of Rogue, unlike other popular modern roguelikes, and it favors simplicity over complexity, while trying to ensure that the interactions  between components will be interesting and varied. It is possible to play entirely with the keyboard, or entirely with the mouse, or a combination of the two as the player chooses.
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">brogue.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://lh3.googleusercontent.com/zPTmFY95Ymh6HuCz-859yCurB9haqSlbJ4FlCkHNofrKzAWY-zVxF_i8FYnOJzfdbzyVDKcjGySkmQgKgUCg-Jd24OG1UDN0Ls-4SQOa9VBmT5cV=w1280</image>
```

Comment:
This link seems not static. It opens for me the first time, now it gives me error 403. I found this [link](https://static.wikia.nocookie.net/brogue/images/6/6a/Brogue-menu.png) for the screenshot from their wiki hosted at Fandom. It is more reliable.